### ISSUE TITLE
SublimeLinter deprecation warning

### ISSUE THREAD

**Issue by externl** [alecthomas/SublimeLinter-contrib-gometalinter#24]
I see this warning consistently every time the linter runs. 

```
gometalinter: live linting file.go in /Users/joe/go/src/path/to/file: /Users/joe/go/bin/gometalinter --fast . -I file.go
SublimeLinter: WARNING: `util.communicate` has been deprecated.
```

**Comment by suryatresna** [comment]
i got same problem, any how to fix it ?

**Comment by asfaltboy** [comment]
The issue is easily fixed by simplifying the calls and using self.communicate in the Linter sub-class. I'll submit a PR (like [here][1])

[1]: https://github.com/fredcallaway/SublimeLinter-contrib-mypy/commit/61af3762a391eaba95cfc2006a7c6615256d16ea
